src-built-in/components/appLauncher2/src/components/AppActionsMenu.jsx

```diff
@@ -30,12 +31,20 @@ export default class AppActionsMenu extends React.Component {
 	}
 
 	//TODO: Implement handlers
-	onAddToFavorite() {
-		const favorite = storeActions.getFolders().find((folder) => {
-			return folder.name === 'Favorites'
-		})
+	async onAddToFavorite() {
+		const favorite = await storeActions.getFavoriteFolder()
```

Comment:
Why is this async?

Reply:
Because storeActions.getFavoriteFolder() returns a promise and we don't want then(function ...)